client/packages/purchasing/src/purchase_order/DetailView/LineEdit/PurchaseOrderLineEditModal.tsx

```diff
@@ -21,22 +23,20 @@ export const PurchaseOrderLineEditModal = ({
   mode,
   isOpen,
   onClose,
+  hasNext,
+  openNext,
 }: PurchaseOrderLineEditModalProps) => {
   const t = useTranslation();
   const { error } = useNotification();
 
-  const lines = purchaseOrder.lines.nodes;
-
-  const [currentLine, setCurrentLine] = useState(
-    lines.find(line => line.id === lineId) ?? undefined
-  );
```

Comment:
This is the bit of state we don't need. We already have the draft line coming in from the hook, no need to store an intermediate copy of it here. 🤷‍♂️